index.html

```diff
@@ -0,0 +1,87 @@
+        </nav>
+    </header>
+
+    <main>
+        <section class="top-section">
+            <div class="top-section-container">
+                <div class="top-section-text">
+                    <p>일상의 모든 물건을 <br>거래해 보세요</p>
+                    <a href="/items.html">구경하러 가기</a>
+                </div>
+                <img src="img/panda-image-top.png" alt="귀여운 판다 한마리가 서있는 이미지">
+            </div>
+        </section>
+        <section class="main-section">
+            <div class="hot-item">
+                <img src="img/main-img-1.png" alt="인기 있는 옷을 바라보는 판다 두마리의 뒷모습" class="hot-item-img">
+                <div class="text-container">
+                    <div class="small-text">Hot item</div>
+                    <div class="big-text">인기 상품을 <br> 확인해 보세요</div>
+                    <p>가장 HOT한 중고거래 물품을 <br> 판다 마켓에서 확인해 보세요</p>
+                </div>
+            </div>
+        </section>
+        <section class="main-section">
+            <div class="search">
+                <div class="text-container">
+                    <div class="small-text">Search</div>
+                    <div class="big-text">구매를 원하는 <br> 상품을 검색하세요</div>
+                    <p>구매하고 싶은 물품은 검색해서 <br> 쉽게 찾아보세요</p>
+                </div>
+                <img src="img/main-img-2.png" alt="원하는 상품을 검색하는 이미지" class="search-img">
+            </div>
+        </section>
+        <section class="main-section">
+            <div class="register">
+                <img src="img/main-img-3.png" alt="연필, 필통, 도서, 안경 같은 물건을 등록하는 이미지" class="register-img">
+                <div class="text-container">
+                    <div class="small-text">Register</div>
+                    <div class="big-text">판매를 원하는 <br> 상품을 등록하세요</div>
+                    <p>어떤 물건이든 판매하고 싶은 상품을 <br> 쉽게 등록하세요</p>
+                </div>
+            </div>
+        </section>
+        <section class="bottom-section">
+            <div class="bottom-section-container">
+                <div class="bottom-section-text">
+                    <p>믿을 수 있는 <br> 판다마켓 중고 거래</p>
+                </div>
+                <img src="img/panda-image-bottom.png" alt="귀여운 판다 두마리가 중고 거래하는 이미지">
+            </div>
+        </section>
+    </main>
+
+    <footer>
+        <div class="codeit">©codeit - 2024</div>
+        <div class="footer-center">
+            <a href="/privacy.html" class="privacy-policy"> Privacy Policy </a>
+            <a href="/faq.html" class="faq"> FAQ </a>
+        </div>
+        <div class="footer-right">
+            <a href="https://www.facebook.com" target="_blank"><img src="img/facebook-icon.png" alt="페이스북 로고"></a>
+            <a href="https://www.twitter.com" target="_blank"><img src="img/twitter-icon.png" alt="트위터 로고"></a>
+            <a href="https://www.youtube.com" target="_blank"><img src="img/youtube-icon.png" alt="유튜브 로고"></a>
+            <a href="https://www.instagram.com" target="_blank"><img src="img/instagram-icon.png" alt="인스타그램 로고"></a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 잇습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-right">
          <li>
            <a href="https://www.facebook.com" target="_blank"><img src="img/facebook-icon.png" alt="페이스북 로고"></a>
          </li>
        </ul>
```

[mdn](https://developer.mozilla.org/ko/docs/Glossary/Semantics#html_%EC%8B%9C%EB%A7%A8%ED%8B%B1)